### PR TITLE
Proofreading and additions to upgrading guide

### PR DESCRIPTION
**Pull request by jenweber**
I over-optimistically named this branch mvp. It still needs a little work to remove unfinished pages from the `pages.yml` or add a banner to them that says "help us with this page"

### CODE REVIEW

File: guides/basic-use/index.md [hunk 1]
```diff
@@ -20,21 +22,28 @@ This will make the `ember` command available throughout your project folders. Th
 
 ### Additional steps for Mac and Linux users
 
-For Mac and Linux users we recommend installing [Watchman](https://facebook.github.io/watchman/). Watchman helps correct for some buggy and inefficient file watching behavior. Do not use the `npm` package by the same name.
+For Mac and Linux users, we recommend installing [Watchman](https://facebook.github.io/watchman/). Do not use the `npm` package by the same name.
+
+Why is Watchman needed?
```
Comment (Gaurav0) [inline guides/basic-use/index.md]: This formats as part of same paragraph as below, should be formatted separately with emphasis

File: guides/basic-use/index.md [hunk 2]
```diff
@@ -20,21 +22,28 @@ This will make the `ember` command available throughout your project folders. Th
...
+When an Ember developer is running a server locally, the server should
+automatically and quickly rebuild when files change.
+However, for some versions of Mac or Linux, this auto-restarting
+does not work perfectly, due to problems in the operating system's
+built-in file watching tools. Watchman helps correct for that buggy and inefficient file watching behavior.
```
Comment (Gaurav0) [inline guides/basic-use/index.md]: Try to separate sentences on separate lines when possible.

File: guides/basic-use/upgrading.md [hunk 3]
```diff
@@ -20,9 +20,25 @@ ember --version
...
+
+1. Moving from patch versions (like 3.4.1 to 3.4.2) will not change the app's behavior. They are mostly bugfixes and internal refactors. These upgrades are usually very quick and require no changes besides changing some dependency versions in `package.json`
+2. Upgrading minor versions (like 3.4 to 3.5) will give access to new features, but the codebase will not need to change, so these are also usually very quick. Afterwards, developers may see new notices about deprecations, whether from their own app or addons they are using. Deprecations are warnings that some API or behavior will be changed in the future for a major release (like Ember 2 to 3)
+2. Major version upgrades (like Ember 2 to 3) introduce breaking changes. Before the upgrade can be made, code inside the app may need to change. See [Managing major upgrades](#managing-major-upgrades) below for the overall strategy.
+
+Developers who upgrade infrequently are encouraged to choose LTS or "long term support" versions of Ember, which receive security updates for an extended time after release. See the [Ember release blog posts](https://www.emberjs.com/blog/tags/releases.html) to find the latest LTS version.
```
Comment (Gaurav0) [inline guides/basic-use/upgrading.md]: Developers who find it challenging to upgrade minor versions as frequently as Ember does

Comment (jenweber) [reply guides/basic-use/upgrading.md]: nice, I like this wording a lot

**Comment by jenweber** [comment]
Thanks @Gaurav0 for the review! I made the edits you suggested. Merged.

**Pull request merged**